helpers/helpers.v1.d/hardware

```diff
@@ -103,3 +103,109 @@ ynh_require_ram() {
         return 0
     fi
 }
+
+# Add swap
+#
+# usage: ynh_add_swap --size=SWAP in MiB
+# | arg: -s, --size= - Amount of SWAP to add in MiB.
+ynh_add_swap () {
```

Comment:
We need an additional helper, that would "just" call `yunohost settings {get,set} 'swap.swapfile.swapfile_size'` and make `new_swap_size = current_swap_size + requested_app_swap_size`.